src/main/scala/io/iohk/ethereum/db/dataSource/DataSource.scala

```diff
@@ -0,0 +1,26 @@
+package io.iohk.ethereum.db.dataSource
+
+trait DataSource {
+
+  type Key = Array[Byte]
+  type Value = Array[Byte]
```

Comment:
(If you don't need ByteString in your type, you can use IndexedSeq[Byte] as ByteString is a IndexedSeq[Byte])